### ISSUE TITLE
Uncaught Exception: Variable "severity Message" does not exist

### ISSUE THREAD

**Issue by bmcarvalho** [PrestaShop/PrestaShop#23588]
#### Describe the bug

After upgrading to 1.7.7.2 everytime some one logs in to backoffice email is sent to owner reporting error log

![image](https://user-images.githubusercontent.com/80357797/110543298-0a808c80-8122-11eb-8f72-2b94d98c7058.png)

Severity should be 1-5 and not 127

 

when debuging and trying to accces log agains i get the following error

Variable "severityMessage" does not exist.

 

```
CRITICAL10:59:44phpUncaught Exception: Variable "severityMessage" does not exist.

CRITICAL10:59:44requestUncaught PHP Exception Twig\Error\RuntimeError: "Variable "severityMessage" does not exist." at /home/osotaoda/public_html/src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/severity_level.html.twig line 47

Hide context    Hide trace

[▼ "exception" => RuntimeError {#3428 ▶} ]
```
 
LOGS : 

```
2021-03-09 19:45:33] request.INFO: Matched route "admin_module_notification_count". {"route":"admin_module_notification_count","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\Modules\\AlertsController::notificationsCountAction","_legacy_controller":"AdminModulesNotifications","_legacy_link":"AdminModulesNotifications:count","_route":"admin_module_notification_count"},"request_uri":"https://www.xxxxxxxxxx.pt/adminfolder/index.php/improve/modules/alerts/count?_token=nIndVuIfQffYv1_9eA0MiSY5JYM13A5WqWZT49lbDVo","method":"GET"} []
[2021-03-09 19:45:33] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-03-09 19:45:33] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"xxxxxxxxxxxx"} []
[2021-03-09 19:45:33] app.ERROR: Data from PrestaShop Addons is invalid, and cannot fallback on cache.  {"exception":"Client error response [url] https://api-addons.prestashop.com?format=json&iso_lang=pt&iso_code=PT&version=1.7.7.2&shop_url=http%3A%2F%2Fxxxxxxxxxxxxxxxxxxxx&method=listing&action=customer&username=xxxxxxxxx&password=xxxxxxxx [status code] 400 [reason phrase] Bad Request"} []
[2021-03-09 19:45:33] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
```
 **DEV.log**

```
[2021-03-09 12:44:57] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-03-09 12:44:58] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 12:44:58] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"5cb2a5","_route":"_wdt"},"request_uri":"https://xxxxxxxxxxxx/adminfolder/index.php/_wdt/5cb2a5?_token=g-EjLmbA-GLH5nnQotvn0NCG_xvYuEppCnWDuDym27U","method":"GET"} []
[2021-03-09 12:44:58] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/xxxxxxx/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 12:44:58] php.INFO: User Deprecated: Implementing "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface" without the "reset()" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class "Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector". {"exception":"[object] (ErrorException(code: 0): User Deprecated: Implementing \"Symfony\\Component\\HttpKernel\\DataCollector\\DataCollectorInterface\" without the \"reset()\" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class \"Csa\\Bundle\\GuzzleBundle\\DataCollector\\GuzzleCollector\". at /home/xxxxxxxx/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/Profiler.php:215)"} []
[2021-03-09 12:44:58] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 12:44:58] request.INFO: Matched route "admin_common_notifications". {"route":"admin_common_notifications","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\CommonController::notificationsAction","_legacy_controller":"AdminCommon","_legacy_link":"AdminCommon","_route":"admin_common_notifications"},"request_uri":"https://xxxxxxxxxx/adminfolder/index.php/common/notifications?_token=g-EjLmbA-GLH5nnQotvn0NCG_xvYuEppCnWDuDym27U&rand=1615293898797","method":"POST"} []
[2021-03-09 12:44:58] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-03-09 12:44:58] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"xxxxxxxxxxxxxxx"} []
[2021-03-09 12:44:58] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/osotaoda/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 12:44:58] php.INFO: User Deprecated: Implementing "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface" without the "reset()" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class "Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector". {"exception":"[object] (ErrorException(code: 0): User Deprecated: Implementing \"Symfony\\Component\\HttpKernel\\DataCollector\\DataCollectorInterface\" without the \"reset()\" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class \"Csa\\Bundle\\GuzzleBundle\\DataCollector\\GuzzleCollector\". at /home/xxxxxxxx/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/Profiler.php:215)"} []
[2021-03-09 12:44:58] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-03-09 12:45:04] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 12:45:04] request.INFO: Matched route "admin_performance_save". {"route":"admin_performance_save","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\PerformanceController::processFormAction","_legacy_controller":"AdminPerformance","_legacy_link":"AdminPerformance:update","_route":"admin_performance_save"},"request_uri":"https://xxxxxxxx/adminfolder/index.php/configure/advanced/performance/?_token=g-EjLmbA-GLH5nnQotvn0NCG_xvYuEppCnWDuDym27U","method":"POST"} []
[2021-03-09 12:45:04] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-03-09 12:45:04] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"xxxxxxxxxxxxxxx"} []
[2021-03-09 12:45:04] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/osotaoda/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 12:45:04] doctrine.DEBUG: SELECT * FROM pshu_memcached_servers [] []
[2021-03-09 12:45:04] php.INFO: User Deprecated: Implementing "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface" without the "reset()" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class "Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector". {"exception":"[object] (ErrorException(code: 0): User Deprecated: Implementing \"Symfony\\Component\\HttpKernel\\DataCollector\\DataCollectorInterface\" without the \"reset()\" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class \"Csa\\Bundle\\GuzzleBundle\\DataCollector\\GuzzleCollector\". at /home/xxxxxx/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/Profiler.php:215)"} []
[2021-03-09 12:45:04] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
```
 
---

```
[2021-03-09 22:37:44] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 22:37:44] request.INFO: Matched route "admin_performance". {"route":"admin_performance","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\PerformanceController::indexAction","_legacy_controller":"AdminPerformance","_legacy_link":"AdminPerformance","_route":"admin_performance"},"request_uri":"https://www.xxxxxxxx.pt/adminfolder/index.php/configure/advanced/performance/?_token=LlncHPse5mRKlZ66lG2yRvccAoFUEJlVqMphD-BXhMA","method":"GET"} []
[2021-03-09 22:37:44] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-03-09 22:37:44] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"xxxxxxxxxxx"} []
[2021-03-09 22:37:44] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/osotaoda/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 22:37:44] doctrine.DEBUG: SELECT * FROM pshu_memcached_servers [] []
[2021-03-09 22:37:44] doctrine.DEBUG: SELECT * FROM pshu_memcached_servers [] []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Product Comments","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Allows users to post reviews and rate products on specific criteria.","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Sell your first product quicker than you would have wished with our nice onboarding process.","domain":"ModulesWelcomeAdmin","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Merchant Expertise","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Become an e-commerce expert within the blink of an eye!","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Google Ads","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Get visible results with Shopping Ads from Google. Make it easy for shoppers to discover and buy your products with a Google Smart Shopping campaign in PrestaShop.","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Moloni","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Payment With Fee","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Set extra fee for any payment method such as cash on delivery (COD), bank wire, Paypal, Stripe, etc. Create unlimited number of custom payment method with/without fee","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"General","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Visibility","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Google Analytics","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Gain clear insights into important metrics about your customers, using Google Analytics","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:44] translation.WARNING: Translation not found. {"id":"Are you sure you want to uninstall Google Analytics? You will lose all the data related to this module.","domain":"messages","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"Manage your quick accesses","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"Product name, reference, etc.","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"Resources","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"https://www.prestashop.com/en/training?utm_source=back-office&utm_medium=profile&utm_campaign=training-en&utm_content=download17","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"https://www.prestashop.com/en/experts?utm_source=back-office&utm_medium=profile&utm_campaign=expert-en&utm_content=download17","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"Find an Expert","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"https://addons.prestashop.com?utm_source=back-office&utm_medium=profile&utm_campaign=addons-en&utm_content=download17","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] translation.WARNING: Translation not found. {"id":"https://www.prestashop.com/en/contact?utm_source=back-office&utm_medium=profile&utm_campaign=help-center-en&utm_content=download17","domain":"AdminNavigationHeader","locale":"pt-PT"} []
[2021-03-09 22:37:45] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-03-09 22:37:46] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 22:37:46] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"1c7088","_route":"_wdt"},"request_uri":"https://www.xxxxxx.pt/adminfolder/index.php/_wdt/1c7088?_token=LlncHPse5mRKlZ66lG2yRvccAoFUEJlVqMphD-BXhMA","method":"GET"} []
[2021-03-09 22:37:46] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/osotaoda/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 22:37:46] php.INFO: User Deprecated: Implementing "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface" without the "reset()" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class "Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector". {"exception":"[object] (ErrorException(code: 0): User Deprecated: Implementing \"Symfony\\Component\\HttpKernel\\DataCollector\\DataCollectorInterface\" without the \"reset()\" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class \"Csa\\Bundle\\GuzzleBundle\\DataCollector\\GuzzleCollector\". at /home/osotaoda/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/Profiler.php:215)"} []
[2021-03-09 22:37:47] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 22:37:47] request.INFO: Matched route "admin_common_notifications". {"route":"admin_common_notifications","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\CommonController::notificationsAction","_legacy_controller":"AdminCommon","_legacy_link":"AdminCommon","_route":"admin_common_notifications"},"request_uri":"https://www.xxxxxx.pt/adminfolder/index.php/common/notifications?_token=LlncHPse5mRKlZ66lG2yRvccAoFUEJlVqMphD-BXhMA&rand=1615329467828","method":"POST"} []
[2021-03-09 22:37:47] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-03-09 22:37:47] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"xxxxxxxx"} []
[2021-03-09 22:37:47] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/osotaoda/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 22:37:47] php.INFO: User Deprecated: Implementing "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface" without the "reset()" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class "Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector". {"exception":"[object] (ErrorException(code: 0): User Deprecated: Implementing \"Symfony\\Component\\HttpKernel\\DataCollector\\DataCollectorInterface\" without the \"reset()\" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class \"Csa\\Bundle\\GuzzleBundle\\DataCollector\\GuzzleCollector\". at /home/osotaoda/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/Profiler.php:215)"} []
[2021-03-09 22:37:47] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-03-09 22:37:56] doctrine.DEBUG: SELECT name FROM pshu_module WHERE active = 1 [] []
[2021-03-09 22:37:56] request.INFO: Matched route "admin_performance_save". {"route":"admin_performance_save","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\PerformanceController::processFormAction","_legacy_controller":"AdminPerformance","_legacy_link":"AdminPerformance:update","_route":"admin_performance_save"},"request_uri":"https://www.xxxxxx.pt/adminfolder/index.php/configure/advanced/performance/?_token=LlncHPse5mRKlZ66lG2yRvccAoFUEJlVqMphD-BXhMA","method":"POST"} []
[2021-03-09 22:37:56] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-03-09 22:37:56] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"xxxxxx.pt"} []
[2021-03-09 22:37:56] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /home/osotaoda/public_html/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-03-09 22:37:56] doctrine.DEBUG: SELECT * FROM pshu_memcached_servers [] []
[2021-03-09 22:37:57] php.INFO: User Deprecated: Implementing "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface" without the "reset()" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class "Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector". {"exception":"[object] (ErrorException(code: 0): User Deprecated: Implementing \"Symfony\\Component\\HttpKernel\\DataCollector\\DataCollectorInterface\" without the \"reset()\" method is deprecated since Symfony 3.4 and will be unsupported in 4.0 for class \"Csa\\Bundle\\GuzzleBundle\\DataCollector\\GuzzleCollector\". at /home/osotaoda/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/Profiler.php:215)"} []
[2021-03-09 22:37:57] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []

```

#### Additional information

* PrestaShop version: 1.7.7.2
* PHP version: 7.2

**Comment by hibatallahAouadni** [comment]
Hello @bmcarvalho 

Could you please provide your previous PrestaShop version and the 1-click upgrade module version.
Cause I just check with three shops and I didn't encounter your issue:
- 1.7.7.0 upgraded to 1.7.7.2: no problem
- 1.7.6.4 upgraded to 1.7.7.2: no problem
- 1.7.4.1 upgraded to 1.7.7.2: no problem

![image](https://user-images.githubusercontent.com/23308427/110603496-5d405f80-8187-11eb-9a98-cb46a65f7c08.png)

Please check and feedback.
Thanks!

**Comment by bmcarvalho** [comment]
Good morning

Always using last stable version

Updated from 1.7.7.0 (last stable ) to 1.7.7.2

1 click upgrade : V 4.12.0


best regards

Bruno Carvalho

**Comment by hibatallahAouadni** [comment]
Hello @bmcarvalho 

As I mentioned above I upgraded my shop from 1.7.7.0 to 1.7.7.2 and I didn't encounter your issue.
Did you encounter any issue while upgrading?

Thanks!

**Comment by bmcarvalho** [comment]
good morning

During the update process no issue. started to receive the log emails on the first log in after the update

correction updated from 1.7.7.1  to 1.7.7.2.

now all shop just crashed. will try to restore last backup and update again

Best Regards

Bruno Carvalho

**Comment by hibatallahAouadni** [comment]
Hello @bmcarvalho 

Even when I upgrade my shop from 1.7.7.1 to 1.7.7.2 I didn't encounter your issue:

![image](https://user-images.githubusercontent.com/23308427/110657291-02c3f500-81c1-11eb-80fb-e1bed8a745c7.png)

Please check and feedback.
Thanks!

**Comment by bmcarvalho** [comment]
Went back for last backup and all running withouth error
today after updating ps metrics modules all started again


Protect vendor folder in module ps_metrics
Erro 127 internal server error 
Erro (3) | [PS_METRICS] Unable to decode Json


unistalled PS metrics module and no more errors.

Any suggestion ?


running : 1.7.7.2

**Comment by florine2623** [comment]
Hello @bmcarvalho ,

I can't reproduce your issue. I tried upgrading from 1.7.2.4 to 1.7.7.2.
![Capture d’écran 2021-03-25 à 11 40 44](https://user-images.githubusercontent.com/16019289/112460207-f1512000-8d5e-11eb-8c9e-15dcee0d376c.png)

If you have an issue with PS Metrics, you should contact PS Metrics Support directly.

Despite our efforts, we were not able to reproduce your issue with the provided information. It seems that your issue is not a PrestaShop core bug but most likely a server configuration or customization problem.
We will close this issue.

Thanks!

**Comment by Prestaworks** [comment]
This seems to be related to turning the debug mode on and off.

**Comment by matks** [comment]
@bmcarvalho @Prestaworks PR merged https://github.com/PrestaShop/PrestaShop/pull/23856

this will be fixed in 1.7.8

**Comment by ipierre1** [comment]
> Went back for last backup and all running withouth error
> today after updating ps metrics modules all started again
> 
> Protect vendor folder in module ps_metrics
> Erro 127 internal server error
> Erro (3) | [PS_METRICS] Unable to decode Json
> 
> unistalled PS metrics module and no more errors.
> 
> Any suggestion ?
> 
> running : 1.7.7.2

The problem seems to come from PS_Metrics, indeed I also had the problem with the logs and a lot of mails sent by PS. Once deactivated, no more problems ... 
But the strangest thing is that there have been no changes that had been made on the site itself for a few months (PS version 1.7.6.9 and PS_Metric was up to date)
